scalding-core/src/main/scala/com/twitter/scalding/reducer_estimation/InputSizeReducerEstimator.scala

```diff
@@ -59,4 +59,15 @@ class InputSizeReducerEstimator extends ReducerEstimator {
 
         Some(nReducers)
     }
+  }
+
 }
+
+// TODO: can we delete this? Just make the object above extend ReducerEstimator?
```

Comment:
we use reflection to create these. Why not just leave it as is? Sometimes people get confused with reflection and objects because their name (due to mangling) is not always clear.